src/kp2bw/bw_serve.py

```
@@ -0,0 +1,583 @@
+                for filename, data in attachments:
+                    tasks.append(
+                        asyncio.create_task(
+                            _upload_one(client, item_id, filename, data)
+                        )
+                    )
+            results = await asyncio.gather(*tasks, return_exceptions=True)
+
+        failures = [r for r in results if isinstance(r, BaseException)]
+        if failures:
+            for err in failures:
+                logger.error(f"Attachment upload failed: {err}")
```

Ruff (0.15.2)

[warning] Logging statement uses f-string

(G004)